content/docs/rfcs/33/README.md

```diff
@@ -0,0 +1,122 @@
+---
+slug: 33
+title: 33/WAKU2-DISCV5
+name: WAKU v2 Discv5 Ambient Peer Discovery
+status: raw
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# Abstract
+
+`33/WAKU2-DISCV5` specifies a modified version of [Ethereum's Node Discovery Protocol v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) as a means for ambient node discovery.
+[10/WAKU2](/specs/10) uses the `33/WAKU2-DISCV5` ambient node discovery network for establishing a decentralized network of interconnected Waku2 nodes.
+The `33/WAKU2-DISCV5` discovery network is isolated from the Ethereum Discovery v5 network; this isolation greatly improves discovery efficiency.
+
+# Background and Rationale
+
+[11/WAKU2-RELAY](/specs/11) assumes the existence of a network of Waku2 nodes.
+For establishing and growing this network, new nodes trying to join the Waku2 network need a means of discovering nodes within the network.
+[10/WAKU2](/specs/10) supports the following discovery methods in order of increasing decentralization
+
+* hard coded bootstrap nodes
+* `DNS discovery`
+* `peer-exchange` protocol
+* `33/WAKU2-DISCV5` (specified in this document)
+
+The purpose of ambient node discovery within [10/WAKU2](/specs/10) is discovering Waku2 nodes in a decentralized way.
+The unique selling point of `33/WAKU2-DISCV5` is its holistic view of the network, which allows avoiding hotspots and allows merging the network after a split.
+While the other methods provide either a fixed or local set of nodes, `33/WAKU2-DISCV5` can provide a random sample of Waku2 nodes.
+Future iterations of this document will add the possibility of efficiently discovering Waku2 nodes that have certain capabilities, e.g. holding messages of a certain time frame during which the querying node was offine.
+
+## Separate Discovery Network
+
+### w.r.t. Waku2 Relay Network
+
+`33/WAKU2-DISCV5` spans an overlay network separate from the [GossipSub](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md) network [11/WAKU2-RELAY](/specs/11) builds on.
+Being a P2P network on its own, it also depends on bootstrap nodes.
```

Comment:
Since we still need bootstrap nodes for a separate discovery network, wouldn't it be somewhat centralized? (similar to the discovery method#1  you listed above).  How about a hybrid approach where some of the nodes in the waku discovery network also take part in the Eth discv5 net hence be discoverable there as well?